xen/arch/x86/setup.c

```diff
@@ -1928,6 +1962,14 @@ void __init noreturn __start_xen(unsigned long mbi_p)
                "Multiple initrd candidates, picking module #%u\n",
                initrdidx);
 
+    sl_tpm = enable_tpm();
+    extend_pcr(sl_tpm, (void*)(long)mod->mod_start,
+               mod->mod_end - mod->mod_start, 0, 0);
+
+    intrid = (module_t*)(initrdidx < mbi->mods_count ? mod + initrdidx : NULL);
```

Comment:
```suggestion
    initrid = (module_t*)(initrdidx < mbi->mods_count ? mod + initrdidx : NULL);
```

A typo in variable name?

Reply:
Yes, I added a new variable to not repeat this if/else statement. Named it wrong at first and then reused it. Changed it everywhere like you suggested here.